packages/web-components/src/counter-badge/counter-badge.template.ts

```diff
@@ -1,10 +1,11 @@
 import { ElementViewTemplate, html } from '@microsoft/fast-element';
 import { badgeTemplate } from '../badge/badge.template.js';
+import { Badge } from '../badge/badge.js';
 import { CounterBadge } from './counter-badge.js';
 import { CounterBadgeOptions } from './counter-badge.options.js';
 
-function composeTemplate<T extends CounterBadge>(options: CounterBadgeOptions = {}): ElementViewTemplate<T> {
-  return badgeTemplate({
+function composeTemplate<T extends CounterBadge & Badge>(options: CounterBadgeOptions = {}): ElementViewTemplate<T> {
```

Comment:
`CounterBadge` composes `Badge` template but does not inherit from it.
@chrisdholt Are you OK with this change or would you propose a different fix?

Reply:
Let's leave it for now and see how it goes